docs/config.md

```diff
@@ -1,6 +1,6 @@
 ## Config format
 
-This document provides examples for the .yaml format for specifying rules to sgrep. It does not cover the patterns themselves; for information on the query sytnax see [patterns.md](patterns.md)
+This document provides examples for the .yaml format for specifying rules to sgrep. It does not cover the patterns themselves; for [the pattern syntax see patterns.md](patterns.md)
```

Comment:
This config doc does more than provide examples: it also defines/describes operators, fields, and their composition. Thus I’d suggest something like:

```suggestion
This document describes sgrep rule fields and provides rule examples. It does not cover the patterns themselves; for [the pattern syntax see patterns.md](patterns.md)
```

I wonder if the content for patterns.md should be merged into this doc? I see that patterns.md is rather long and that might be a challenge, but they seem nearly inseparable (can’t have a rule without a pattern, and you can’t have a pattern without a rule).

Reply:
@pabloest agreed they should probably be unified eventually, but don't see an easy way to do that now